src/Symfony/Component/ErrorHandler/Error/ClassNotFoundFatalError.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Error;
+
+class ClassNotFoundFatalError extends ClassNotFoundError implements FatalErrorInterface
```

Comment:
This behavior is necessary because of https://3v4l.org/KjaLl (we can get rid of it once min PHP requirement is > 7.3)

Reply:
You cannot get rid of it, as PHP 7.4 also needs it according to your snippet.

Reply:
You are right.

Reply:
It's actually back on 7.4.